core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -255,7 +255,7 @@ oppia.controller('StatisticsTab', [
                 htmlSnippets = visualizationsInfo.map(
                   function(visualizationInfo) {
                     var isAddressedResults = null;
-                    if (visualizationInfo.options.show_addressed_info) {
+                    if (visualizationInfo.options.use_addressed_info) {
```

Comment:
I think show_* is better, FWIW. "use" implies that the info is already there.

Reply:
Done.